Sources/Routes/Configuration/Routes.swift

```diff
@@ -1,34 +1,77 @@
-import Foundation
 import ServerIO
```

Comment:
This is definitely a _BIG_ improvement. I think you could potentially even use one data structure for all this information.
```
Routes = [
                     ["routePath": .root, "routeHandler": Root(), "allowedActions": [.get, .head]],
                     ["routePath": .hello, "routeHandler": Hello(), "allowedActions": [.get, .head]],
                     ["routePath": .tea, "routeHandler": Tea(), "allowedActions": [.get, .head]],
                     etc...
        ]
``` 
Curious what you think about that? I'm not saying it has to happen or even that it's definitely better... This file just seems like it contains a lot of closely related information that's really spread out.